addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -593,6 +601,17 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
                 return this.activeForm.id;
             case 'addActionField': {
                 const value = this.$target.find(`.s_website_form_dnone input[name="${params.fieldName}"]`).val();
+                if (params.fieldName === 'email_to') {
+                    // For email_to, we try to find a value in this order:
+                    // 1. The current value of the input
+                    // 2. The data-for value if it exists
+                    // 3. The default value (`defaultEmailToValue`)
+                    const defaultEmailToValue = "info@yourcompany.example.com";
+                    if (value && value !== defaultEmailToValue) {
+                        return value;
+                    }
+                    return this.dataForEmailTo || defaultEmailToValue;
+                }
```

Comment:
This is not right: if you empty the input, you have `value` which is falsy, which ends up showing the dataFor value... although sending the form does not actually send any email as no `email_to` is set. I think this code could be left as it is if you make that feature work: if you empty the input, this restores `info@yourcompany.example.com`

Reply:
> What do you mean

Make it so emptying the input is equal to coming back to the default configuration of the form, which I am thinking is:
- `info@yourcompany.example.com` set up as static value (am I wrong on that one?)
- but data-for used when sending the form
- but data-for displayed in the right panel

Reply:
> > although sending the form does not actually send any email as no email_to is set
>
> This PR is only about fixing the right panel email_to

@rdeodoo You forgot the most important part when quoting me: "if you empty the input" :wink: 

> That shouldn't restore this default value but the one which will be used by the form: the data-for one, right?

Is not this what I said in my last message? :)

> > Make it so emptying the input is equal to coming back to the default configuration of the form, which I am thinking is:
> >
> > - info@yourcompany.example.com set up as static value (am I wrong on that one?)
> > - **but data-for used when sending the form**
> > - **but data-for displayed in the right panel**

Reply:
@rdeodoo To be clearer:

- Before this PR: if you empty the email_to input, nothing is shown in the input. After save, the form sends no email if filled.
- After this PR: if you empty the email_to input, it automatically gets filled with the data-for email (cool!). But after save, the form sends no email if filled (omg, terrible).

Reply:
Perfect :+1: 
Didn't want to disturb the review process, it was just to be sure.